src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionTest {
+    private static final String APPLE = "apple";
+    private static final int INITIAL_FRUIT_QUANTITY = 5;
+    private FruitTransaction fruitTransaction;
+
+    @BeforeEach
+    void setUp() {
+        fruitTransaction = new FruitTransaction();
+    }
+
+    @Test
+    void setAndGetAllOperations_Ok() {

```

Comment:
please read task description / common mistakes again and try following naming convention for test methods